<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MFinishForm</class>
 <widget class="QWidget" name="MFinishForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="info">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>581</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string>感谢使用</string>
   </property>
  </widget>
  <widget class="QToolButton" name="bilibiliButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>400</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Bilibili</string>
   </property>
  </widget>
  <widget class="QToolButton" name="fileButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>400</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>查看导出的文件</string>
   </property>
  </widget>
  <widget class="QToolButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>570</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>退出软件</string>
   </property>
  </widget>
  <widget class="QToolButton" name="githubButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>400</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>GitHub</string>
   </property>
  </widget>
  <widget class="QToolButton" name="beginButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>570</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>返回开始界面</string>
   </property>
  </widget>
  <widget class="QToolButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>560</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>返回编辑界面</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
